src/test/java/io/phasetwo/service/resource/OrganizationResourceTest.java

```diff
@@ -1,71 +1,267 @@
+
+    assertThat(rep, notNullValue());
+    assertThat(rep.getId(), notNullValue());
+    assertThat(rep.getDisplayName(), CoreMatchers.nullValue());
+    assertThat(rep.getUrl(), CoreMatchers.nullValue());
+    assertThat(rep.getRealm(), is(REALM));
+    assertThat(rep.getDomains().iterator().next(), is("example.com"));
+    assertThat(rep.getName(), is("example"));
+    assertThat(rep.getId(), is(id));
+
+    // get list
+    Response response = getRequest();
+    assertThat(response.getStatusCode(), is(jakarta.ws.rs.core.Response.Status.OK.getStatusCode()));
+    List<OrganizationRepresentation> organizations = new ObjectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+    assertNotNull(organizations);
+    assertThat(organizations.size(), is(1));
+
+    rep = organizations.get(0);
+    assertThat(rep.getId(), notNullValue());
+    assertThat(rep.getDisplayName(), CoreMatchers.nullValue());
+    assertThat(rep.getUrl(), CoreMatchers.nullValue());
+    assertThat(rep.getRealm(), is(REALM));
+    assertThat(rep.getDomains().iterator().next(), is("example.com"));
+    assertThat(rep.getName(), is("example"));
+    assertThat(rep.getId(), is(id));
 
+    // update
+    rep.url("https://www.example.com/").displayName("Example company").attributes(ImmutableMap.of("foo", List.of("bar")));
+
+    response = putRequest(rep, id);
+    assertThat(response.statusCode(), is(jakarta.ws.rs.core.Response.Status.NO_CONTENT.getStatusCode()));
+
+    // get single
+    response = getRequest(id);
+    rep = new ObjectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+    assertThat(response.getStatusCode(), is(jakarta.ws.rs.core.Response.Status.OK.getStatusCode()));
+
+    assertThat(rep.getId(), notNullValue());
+    assertThat(rep.getAttributes(), notNullValue());
+    assertThat(rep.getAttributes().keySet(), hasSize(1));
+    assertThat(rep.getAttributes().get("foo"), hasSize(1));
+    assertThat(rep.getAttributes().get("foo").get(0), is("bar"));
+    assertThat(rep.getDisplayName(), is("Example company"));
+    assertThat(rep.getUrl(), is("https://www.example.com/"));
+    assertThat(rep.getRealm(), is(REALM));
+    assertThat(rep.getDomains().iterator().next(), is("example.com"));
+    assertThat(rep.getName(), is("example"));
+    assertThat(rep.getId(), is(id));
+
+    // delete
+    deleteOrganization(id);
+
+    // get single
+    response = getRequest(id);
+    assertThat(response.getStatusCode(), is(jakarta.ws.rs.core.Response.Status.NOT_FOUND.getStatusCode()));
+
+    // get list
+    response = getRequest();
+
+    assertThat(response.getStatusCode(), is(jakarta.ws.rs.core.Response.Status.OK.getStatusCode()));
+    organizations = new ObjectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+    assertNotNull(organizations);
+    assertThat(organizations.size(), is(0));
+  }
+
+  /////////////////////////////////////
+  // port from OrganizationProviderTest
```

Comment:
I'd like to make sure these test cases are covered here, as we had to remove the `OrganizationProviderTest` with the switch away from using the `testsuite` classes. Can you validate that these are covered by other tests, and delete this if they are, or create new tests if they are not?

Reply:
Thanks for bringing this to my attention. I will review the test cases and validate if they are covered by other tests or not. If they are covered, I will ensure to remove the redundant tests, and if not, I will create new test cases to cover them. I'll keep you updated on the progress and provide a summary once the validation is complete.